src/index.html

```diff
@@ -1,13 +1,300 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;700&family=Space+Mono&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img
+                src="images/logo.png"
+                alt="Bose logo"
+                class="top-bar__logo"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel:+123455555555" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="title header--title">Bring joy back to
+              <br>
+              the everyday
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo">
+            <img
+              src="images/logo.png"
+              alt="Bose logo"
+              class="top-bar__logo"
+            >
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+        <div class="menu__content">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#recommended" class="nav__link">Products</a>
+              </li>
+
+              <li class="nav__item">
+                <a href="#categories" class="nav__link">Store</a>
+              </li>
+
+              <li class="nav__item">
+                <a href="#about-us" class="nav__link">Login</a>
+              </li>
+
+              <li class="nav__item">
+                <a href="#contact-us" class="nav__link">Cart(0)</a>
+              </li>
+
+            </ul>
+          </nav>
+          <a href="tel:+123455555555" class="menu__phone-number">+1 234 5555-55-55</a>
+          <a href="tel:+123455555555" class="menu__call-to-order">call to order</a>
+        </div>
+      </div>
+    </aside>
+    <main class="main">
+      <div class="main__content">
+        <div class="container">
+          <section class="recommended">
+            <h2 class="section-title" id="recommended">Recommended</h2>
+            <div class="recommended__products">
+              <article class="product">
+                <a href="#" class="product__link">
+                  <img
+                    src="./images/Phone (1).png"
```

Comment:
You need to use a name for pictures without spaces in the name, you need to fix that